### PR TITLE
update readme doc

### PR DESCRIPTION
**Pull request by HamadiSE**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,35 +48,32 @@ First, clone this repository:
...
 ```shell-script
-cd cnf-certification-test-partner/test-partner
-oc create -f ./namespace.yaml
-oc create -f ./partner.yaml
+make install-partner-pods
 ```
 
-This will create a Pod named "partner" in the `tnf` namespace.  This Pod is the test partner for running Generic CNF
+
+This will create a Pod named "partner" in the `$TNF_PARTNER_NAMESPACE` namespace.  This Pod is the test partner for running CNF
```
Comment (jun-rh) [inline README.md:63]: pod -> deployment

**Review by jun-rh — CHANGES_REQUESTED** [review]
line 108 to 123 is redundant now?

File: README.md (hunk 1 above)
Comment (HamadiSE) [reply README.md]: fixed

**Comment by HamadiSE** [comment]
> line 108 to 123 is redundant now?

I know it's repeated. The two sections are separate, once to just start partner, the other is to deploy PUT and Partner. I can add it in common section above.

**Review by wying3 — APPROVED** [review]

**Review by jun-rh — APPROVED** [review]

**Pull request merged**